src/HomeComponent/HomeComponent.js

```diff
@@ -4,6 +4,7 @@ import ButtonComponent from './ButtonComponent/ButtonComponent';
 import axios from 'axios';
 import './HomeComponent.css';
 import SearchComponent from './SearchComponent/SearchComponent';
+import { async } from 'q';
```

Comment:
You can remove this because you aren't actually using it anywhere in your file. Adding unnecessary imports increases the size of your file and can make it confusing to understand what you are doing in your code